lib/task_bunny/sync_publisher.ex

```diff
@@ -35,7 +35,9 @@ defmodule TaskBunny.SyncPublisher do
 
   The call is synchronous.
   """
-  @spec push(host :: atom, job :: atom, payload :: any) :: :ok | :failed
+  @spec push(host :: atom, job :: atom | String.t, payload :: any) :: :ok | :failed
```

Comment:
do we need variable names? i'd like to be consistent with the style without variable names in spec.